osquery/tables/system/darwin/certificates.mm

```diff
@@ -18,24 +18,9 @@
 namespace osquery {
 namespace tables {
 
-void genCertificate(const SecCertificateRef& SecCert, QueryData& results) {
+void genCertificate(X509* cert, const std::string path, QueryData& results) {
```

Comment:
We should pass `path` by reference:
```
void genCertificate(X509* cert, const std::string& path, QueryData& results)
```

The alternative is to copy (non-const) then `std::move` (invoke move assignment) on line 37. But IMO they are the same, and a pass-by-const-reference is the cleaner and easier to understand approach.